trunner/harness/psh.py

```diff
@@ -52,12 +52,20 @@ class ShellHarness(IntermediateHarness):
 
     """
 
-    def __init__(self, dut: Dut, prompt: str, cmd: Optional[List[str]] = None, prompt_timeout: Optional[int] = -1):
+    def __init__(
```

Comment:
FIX COMMIT MESSAGE !!!! 

(already written twice in this PR, making it a separate comment so it won't be ignored again)

Reply:
Sorry for that, I misunderstood.